Yoast/YoastSEO.js
# Update French transition word list

## Summary

This PR can be summarized in the following changelog entry: Updates French transition word list, props @evoqueio

## Added words
- à cause de cela
- à juste titre
- à l'époque actuelle
- à nouveau
- à partir de là
- à savoir que
- afin d'attirer l'attention sur
- au fur et à mesure
- c'est dans ce but que
- c'est la raison pour laquelle
- c'est pour cela que
- comme on pouvait s'y attendre
- commençons par examiner
- considérons par exemple
- d'autant plus
- de la même manière
- en tout premier lie
- encore une fois
- il est indéniable que
- il est question de
- il est vrai que
- il faut reconnaître que
- il faut souligner que
- il ne faut pas oublier que
- il suffit de prendre pour exemple
- jusqu'ici 
- il y a aussi
- l'aspect le plus important de
-  l'exemple le plus significatif
- mais en réalité
- outre cela
- plus précisément
- plus tard
- pour commencer
- pour marquer la causalité
- pour prendre un autre exemple
- prenons le cas de
- qui plus est
- touchant à
- tout d'abord examinons
- tout d'abord il faut dire que
- tout en reconnaissant que
- une fois de plus

## Deleted words
- en concequence (was already included with the correct spelling: 'en conséquence')
- quant même (was already included with the correct spelling: 'quand même')

## Fixed mistakes 
- d'une facon générale --> d'une façon générale
- de même qua --> de même qu'à
- étant donné qua --> étant donné qu'a
- la-dessus --> là-dessus
- pur toutes ces raisons --> pour toutes ces raisons

## Test instructions

This PR can be tested by following these steps:

* Use the example
* Write a sentence with one of the newly added words. 
* See whether the sentence is marked as containing a transition word.
* Also check whether deleted and misspelled words are not marked anymore.

Fixes #1175
